File: 11/INFORMATIKA/11KOM2/AP/LAT01/Soal03.java
import java.util.*;

public class Soal03{
    public static void main(String[] args)
    {
        Scanner console = new Scanner(System.in);
        int score = console.nextInt();

        if (score >= 90)
        {
            System.out.println("A");
        } else if (score >= 80)
        {
            System.out.println("B");
        } else if (score >= 70)
        {
            System.out.println("C");
        } else {
            System.out.println("D");
        }
    }
}

File: 10/INFORMATIKA/C++/10-7/07/soal03.cpp
#include <iostream>
using namespace std;

int main()
{
    int score = 89; // B
    if(score >= 90)
    {
        cout << "A";
    } else if (score >= 80 && score <= 89)
    {
        cout << "B";
    } else if (score >= 70 && score <= 79)
    {
        cout << "C";
    } else if (score < 70)
    {
        cout << "D";
    }
}